x/wasm/query_msgs.go

```diff
@@ -0,0 +1,114 @@
+package wasm
+
+import (
+	"encoding/json"
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+
+	coretypes "github.com/sedaprotocol/seda-chain/x/core/types"
+)
+
+const (
+	QueryPathStakerAndSeq      = "/sedachain.core.v1.Query/StakerAndSeq"
+	QueryPathStakingConfig     = "/sedachain.core.v1.Query/StakingConfig"
+	QueryPathDataRequestConfig = "/sedachain.core.v1.Query/DataRequestConfig"
+)
+
+type CoreContractQuery struct {
+	GetStakerAndSeq  *GetStakerAndSeq  `json:"get_staker_and_seq"`
+	GetStakingConfig *GetStakingConfig `json:"get_staking_config"`
+}
+
+type GetStakerAndSeq struct {
+	PublicKey string `json:"public_key"`
+}
+
+type GetStakerAndSeqResponse struct {
```

Comment:
Can't we use the protobuf types for this instead of double creating structs?

`QueryStakerAndSeqResponse` from the protobuf looks similar to this one no?

Reply:
ah nvm